api/serializers/group.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework import serializers
+
+from api.models import Group
+from api.serializers.coordinator import MinifiedCoordinatorSerializer
+from api.serializers.language_and_level import MinifiedLanguageAndLevelSerializer
+from api.serializers.teacher import MinifiedTeacherSerializer
+
+
+class PublicGroupSerializer(serializers.ModelSerializer[Group]):
+    """Representation of a Group that is used in 'All groups' Tooljet view."""
+
+    coordinators = MinifiedCoordinatorSerializer(many=True, read_only=True)
+    teachers = MinifiedTeacherSerializer(many=True, read_only=True)
+    students_count = serializers.IntegerField(read_only=True)
+    language_and_level = MinifiedLanguageAndLevelSerializer(read_only=True)
+
+    class Meta:
+        model = Group
+        fields = (
+            "id",
+            "communication_language_mode",
+            "monday",
+            "tuesday",
+            "wednesday",
+            "thursday",
+            "friday",
+            "saturday",
+            "sunday",
+            "language_and_level",
+            "lesson_duration_in_minutes",
+            "status",
+            "start_date",
+            "end_date",
+            "telegram_chat_url",
+            "coordinators",
+            "teachers",
+            "students_count",  # This field should come from annotation.
```

Comment:
```suggestion
            "students_count",  # This field will be added in ViewSet by annotating a QuerySet.
```
BTW that is a sort of "circular dependency" (serializer knows about view functionality, view imports serializer), but maybe there's no other way.

Reply:
Another option is to use "SerializerMethodField" (or property in model) to count students for each group. It's much slower and inefficient so current solution is better.